chrome/js/onClick.js

```diff
@@ -1,8 +1,15 @@
-
-window.addEventListener("click", function(event) {
-  console.log('in the click WINDOW', event)
+window.addEventListener("click", function (event) {
+  console.log("in the click WINDOW", event);
 });
 
-document.addEventListener("click", function(event) {
-    console.log('were in the click DOCUMENT', event)
+document.addEventListener("click", function (event) {
+  console.log("were in the click DOCUMENT", event);
 });
+
+/**
+ * Returns the highlighted text from a webpage.
+ * @return {string} String of the highlighted text.
+ */
+const grabHighlightedText = () => {
```

Comment:
I think we should consider breaking this out. If you see my diff you'll see I created a separate file and don't need import/exports because content scripts share the same context space and can directly refer to each other.

But i think we should give some thought if we want to liter the content script list or add like an main.js and helpers.js structure